packages/trpc-interface/src/authorize/project.server.ts

```diff
@@ -48,6 +48,15 @@ export const hasProjectPermit = async (
     return true;
   }
 
+  // Allow load webstudiois for clone
+  // @todo rethink permissions to allow everyone has access to various projects
```

Comment:
what is meant by this?

Reply:
We have this edge case that everyone can clone webstudiois project
This check on id doesn't look good. If it will be not an edge-case permission system should be changed somehow so we could say that everyone can have access to project X.

Reply:
Probably every such project would have `everyone` user and for every check we would do additional check for `everyone`

Reply:
The plan is to make new permission for projects which are allowed to be publicly clonable by anyone.

Reply:
created an issue https://github.com/webstudio-is/webstudio-builder/issues/1038